web/react/components/post_info.jsx

```diff
@@ -30,9 +30,7 @@ module.exports = React.createClass({
                         <div className="dropdown">
                         { isOwner || (this.props.allowReply === "true" && type != "Comment") ?
                             <div>
-                                <a href="#" className="dropdown-toggle theme" type="button" data-toggle="dropdown" aria-expanded="false">
-                                    [...]
-                                </a>
+                                <a href="#" className="dropdown-toggle theme" type="button" data-toggle="dropdown" aria-expanded="false" />
```

Comment:
Oh cool. I actually found that part of HTML5 spec kind of annoying with some other stuff I was working on. Merge away then.